packs/zookeeper/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
+zookeeper_hosts: "localhost:2181"
+zookeeper_root: "/"
```

Comment:
Yeah, I set it to "/stormcron" in my config too. I could see needing to override the root path on an action-by-action basis before needing to override the host. But that can be done as a follow-up. :)